ompi/mca/mtl/ofi/mtl_ofi.c

```diff
@@ -14,8 +14,9 @@ OMPI_DECLSPEC extern mca_mtl_ofi_component_t mca_mtl_ofi_component;
 
 mca_mtl_ofi_module_t ompi_mtl_ofi = {
     {
-        8191,        /* max cid - 2^13 - 1 */
-        (1UL << 30), /* max tag value - must allow negatives */
+        (int)((1ULL << MTL_OFI_CID_BIT_COUNT ) - 1), /* max cid */
+        (int)(1ULL << (MTL_OFI_TAG_BIT_COUNT - 2)),/* max tag value - must allow negatives */
+       //         (1UL << 30), /* max tag value - must allow negatives */
```

Comment:
No need to commit a commented-out line like this.

Reply:
>No need to commit a commented-out line like this.

Oops, this slipped out. Will remove 

>I don't think OFI_TAG_BIT_COUNT is right; 

I agree! I asked this in the devel list. I just took a conservative approach and followed what was already there. Let me test this further.